web/src/scripts/actions/tabActions.js

```diff
@@ -75,6 +75,51 @@ export const closeAllTabs = (containerId) => async (dispatch) => {
   dispatch({type: at.CLOSE_ALL_TABS, payload: {containerId}})
 }
 
+export const focusAdjacentTab = (containerId, iterator, options = {}) => async (dispatch, getState) => {
+  const tabs = getState().ui.tabs
+  const { groups, focusedGroupIndex } = _.get(tabs, `${containerId}`)
```

Comment:
Stylistically, I think we agreed on no spaces in the Deco repo: `const {groups, focusedGroupIndex}`. The only exception being imports. Yops and SceneGraph can do whatever. When in doubt, let's just follow what the function above and below are doing